packages/migration-graph-ember/src/entities/ember-app-package.ts

```diff
@@ -12,24 +12,26 @@ export type EmberPackageOptions = PackageOptions;
 
 export class EmberAppPackage extends Package implements IPackage {
   constructor(pathToPackage: string, options: EmberPackageOptions = {}) {
-    super(pathToPackage, options);
-  }
-
-  get excludePatterns(): Array<string> {
-    // TODO Determine ember-config directory from package.json entry
-    const files = [
+    const excludePatterns = [
```

Comment:
What happens if there are other files in the root like `.babelrc.js` or `.eslint.config.js`?

Reply:
We can ignore all `.` dot files.

Reply:
By default it ignores hidden files. Adding a test to validate this.